coremltools/converters/mil/mil/ops/defs/control_flow.py

```diff
@@ -499,6 +626,27 @@ def type_inference(self):
 
 @register_op(doc_str="")
 class list_scatter(Operation):
+    """
+    Scatter ``values`` to ``ls`` at locations ``indices``.
+    
+    Parameters
+    ----------
+    ls: List[*] (Required)
+    
+    indices: <num_updates,i32> (Required)
+        * Indices of ``ls`` to scatter to.
+        * Elements of ``indices`` must be in ``[0, ls.length)`` at runtime.
```

Comment:
Indices can be greater than or equal to length of list. In that case, dynamic resizing of list happens.

Reply:
Is this correct?

>   indices: tensor<num_updates, i32> (Required)
>     	* Indices of ``ls`` to scatter to.
>     	* Elements of ``indices`` must be in ``[0, ls.length)`` at runtime.
>     	* If indices are greater than or equal to the list length, the list is 
>     	dynamically resized.
>